build.mak.in

```diff
@@ -331,7 +331,6 @@ export APP_LDLIBS := $(PJSUA_LIB_LDLIB) \
         $(PJLIB_LDLIB) \
         @LIBS@
 export APP_LDXXLIBS := $(PJSUA2_LIB_LDLIB) \
-        -lstdc++ \
```

Comment:
IIRC there was an issue related to missing `libstdc++` but can't remember the details (which platform, SDK version, etc).
The point is better duplicated than missed? So please make sure this is always duplicated before removing it :)

Reply:
Reopening this. Passed the CI tests, so the patch seems okay.

As for `libstdc++`, it's duplicated here in `PJSUA2_LIB_LDFLAGS`:
https://github.com/pjsip/pjproject/blob/master/pjsip/build/Makefile#L156
which should be more appropriate since it belongs to the pjsua2 C++ library, rather than put it in `build.mak.in`, which applies to the entire library.